src/method/technology.tex

```diff
@@ -0,0 +1,100 @@
+\section{Technology}\label{sec:technology}
+
+Throughout the development of the project, a number of different tools and publicly available packages were used to ease
+the development process.
+These different technologies were chosen based on the team's needs and the requirements of the project.
+The following sections will describe the different technologies used in the project.
+
+\subsection{Programming stack}\label{subsec:programming}
+
+As the chess game would be hosted on a website, and will require multiple users to interact with each other, the team
```

Comment:
Not a good argument for why we use TypeScript. I think it is too indirect. Maybe just say: "As the project is a web application, TypeScript is used".